src/src/hooks/Capabilities.js

```diff
@@ -15,19 +15,21 @@ export function useCapabilities() {
     list.sort((a, b) => a.name.localeCompare(b.name));
   };
 
-  const createCapability = (name, description) => {
+  const createCapability = (name, description, invitations) => {
     addCapability({
       urlSegments: ["capabilities"],
       method: "POST",
       payload: {
         name: name,
         description: description,
+        invitees: invitations,
       },
     });
   };
 
   useEffect(() => {
     if (addedCapability) {
+      console.log(addedCapability);
```

Comment:
remove this :)

Reply:
Addressed in https://github.com/dfds/selfservice-portal/pull/133/commits/cb4b6bfc44c8d7dff3c4ea8e1c66e2dc416c35bd